### ISSUE TITLE
[20.10] Pop OS shell often crashes on Dell XPS 9700

### ISSUE THREAD

**Issue by dodalovic** [pop-os/shell#727]
With the latest update, often when I move my windows (or resize) with the tiling windows enabled, the shell kind of crashes, and all the apps are killed (or closed, no idea)

I'm running it on Dell XPS 9700

**Comment by leviport** [comment]
> @leviport On my dell XPS 9700, with the latest Pop os, I can constantly crash the windows by doing the following:
> 
>     1. Open some app
> 
>     2. press super + s to enter the switch mode (or whatever the name)
> 
>     3. open another app
> 
>     4. now the apps are in the same container, let's say
> 
>     5. Maximize some of two windows (my shortcut is super+m)
> 
>     6. Press, while maximized, super + h / or super + l (to switch left or right to one of two apps)
> 
> 
> This constantly ends up with both apps being killed


Thank you for the detailed instructions. I am also seeing a gnome-shell crash if I do the same thing,. None of my open applications are closed after the gnome-shell restart, but that is probably a minor detail.

**Comment by sitepodmatt** [comment]
This is same bug described here
https://github.com/pop-os/shell/issues/700
tl;dr waiting for ubuntu to package gnome 3.38.2 out

**Comment by sitepodmatt** [comment]
https://wiki.gnome.org/Forty looks like a release tomorrow?
